DiscountDiscover/Models/Deal.h

```diff
@@ -8,6 +8,7 @@
 
 #import <Foundation/Foundation.h>
 #import <CoreLocation/CoreLocation.h>
+#import <GoogleMaps/GoogleMaps.h>
```

Comment:
This model should not be coupled with an external SDK. Instead you could create a `DealMarker` class that has a reference to the deal and another one to the marker.